src/operations/booleans/trees/Node.js

```diff
@@ -0,0 +1,139 @@
+const {plane} = require('../../../math')
+
+// # class Node
+// Holds a node in a BSP tree. A BSP tree is built from a collection of polygons
+// by picking a polygon to split along.
+// Polygons are not stored directly in the tree, but in PolygonTreeNodes, stored in
+// this.polygontreenodes. Those PolygonTreeNodes are children of the owning
+// Tree.polygonTree
+// This is not a leafy BSP tree since there is
+// no distinction between internal and leaf nodes.
+const Node = function (parent) {
+  this.plane = null
+  this.front = null
+  this.back = null
+  this.polygontreenodes = []
+  this.parent = parent
+}
+
+Node.prototype = {
```

Comment:
is this kept as is from the previous code ?
I find a bit of a bummer that we are loosing all history of the files

Reply:
The old repository can be kept, just like Joostn's repository.